tests/acceptance/browzine-primo-adapter-test.js

```diff
@@ -0,0 +1,190 @@
+
+      inject(function ($compile, $rootScope) {
+        $scope = $rootScope.$new();
+
+        $scope.$ctrl = {
+          parentCtrl: {
+            result: {
+              pnx: {
+                display: {
+                  type: ["article"]
+                },
+
+                addata: {
+                  issn: ["0028-4793"],
+                  doi: ["10.1136/bmj.h2575"]
+                }
+              }
+            }
+          }
+        };
+
+        searchResult = $compile(searchResult)($scope);
+      });
+
+      $scope.$ctrl.parentCtrl.$element = searchResult;
+
+      jasmine.Ajax.install();
+
+      primo.searchResult($scope);
+
+      var request = jasmine.Ajax.requests.mostRecent();
+      request.respondWith({
+        status: 200,
+        contentType: "application/json",
+        response: JSON.stringify({
+          "data": {
+            "id": 55134408,
+            "type": "articles",
+            "title": "New England Journal of Medicine reconsiders relationship with industry",
+            "date": "2015-05-12",
+            "authors": "McCarthy, M.",
+            "inPress": false,
+            "availableThroughBrowzine": true,
+            "startPage": "h2575",
+            "endPage": "h2575",
+            "browzineWebLink": "https://browzine.com/libraries/XXX/journals/18126/issues/7764583?showArticleInContext=doi:10.1136/bmj.h2575"
+          },
+          "included": [{
+            "id": 18126,
+            "type": "journals",
+            "title": "theBMJ",
+            "issn": "09598138",
+            "sjrValue": 2.567,
+            "coverImageUrl": "https://assets.thirdiron.com/images/covers/0959-8138.png",
+            "browzineEnabled": true,
+            "browzineWebLink": "https://develop.browzine.com/libraries/XXX/journals/18126"
+          }]
+        })
+      });
+
+      expect(request.url).toMatch(/articles\/doi\/10.1136%2Fbmj.h2575/);
+      expect(request.method).toBe('GET');
+    });
+
+    afterEach(function() {
+      jasmine.Ajax.uninstall();
+    });
+
+    it("should have an enhanced browse article in browzine option", function() {
+      var template = searchResult.find(".browzine");
+      expect(template).toBeDefined();
+      expect(template.text().trim()).toEqual("View Issue Contents");
+      expect(template.find("a.browzine-web-link").attr("href")).toEqual("https://browzine.com/libraries/XXX/journals/18126/issues/7764583?showArticleInContext=doi:10.1136/bmj.h2575");
+      expect(template.find("a.browzine-web-link").attr("target")).toEqual("_blank");
+      expect(template.find("img.browzine-book-icon").attr("src")).toEqual("https://s3.amazonaws.com/thirdiron-assets/images/integrations/browzine_open_book_icon.png");
```

Comment:
Non-blocking comment: we could use assets.thirdiron.com here if we wanted - it’s an alias to the thirdiron-assets bucket. Not a big deal though.